lib/cryptozoologist/dictionaries.rb

```diff
@@ -59,8 +59,8 @@ def create_list(key)
     end
 
     def filter_library(key)
-      dictionaries = library[key].reject do |key, value|
-        Cryptozoologist.configuration.exclude.include?(key)
+      dictionaries = library[key].reject do |k, _v|
+        Cryptozoologist.configuration.exclude.include?(k)
```

Comment:
Nice catch! I am generally not a fan of single letter variables because I find them to be confusing. I would prefer renaming the arg from `key` to something like `filter_key` and keeping the `key, value` in the block (even though `value` is unused)